api/v1alpha1/zz_conversion_test.go

```diff
@@ -76,97 +74,44 @@ func configFuzzer(in *ClusterConfiguration, c fuzz.Continue) {
 		if in.APIServer.Audit.Profile == "" {
 			in.APIServer.Audit.Profile = configv1.DefaultAuditProfileType
 		}
-		in.Items = append(in.Items, runtime.RawExtension{
```

Comment:
Instead of changing these, can you just call populateDeprecated* functions in the fixupHostedCluster functions?
It'd be good to keep this for the HostedControlPlane conversion.